internal/common/enums.rs

```diff
@@ -431,6 +431,18 @@ macro_rules! for_each_enums {
                 /// Scrollbar always visible
                 AlwaysOn,
             }
+
+            // This enum describes the close behaviour of `PopupWindow`
+            enum ClosePolicy {
```

Comment:
I think the enum should also be called `PopupClosePolicy`. Unprefixed it seems a little too generic for me when we don't know of any use outside.

Reply:
> Also no strong feeling.

Ok, strong feeling here though :-)

Plus, you don't have to write this:

```slint
PopupWindow {
    close-policy: PopupClosePolicy.close-on-outside-click;
}
```

You can also write:

```slint
PopupWindow {
    close-policy: close-on-outside-click;
}
```